VERSION

```diff
@@ -1 +1 @@
-v4.6.0
+v4.6.1
```

Comment:
I think we've never formally defined if we should bump the minor or patch version of the release when we bump just the sidecars. My gut feeling is that when at least one sidecar is bumped on the minor version, then I'd also bump the minor version on our release.

Probably a bit of a grey area, but that's my take.

Reply:
Same but from this table it seems like everything 1.27 related should be under 4.6.x. Thus, I'll keep it that way for now.